### PR TITLE
Adds rudimentary support for gene expression querying

### PR DESCRIPTION
**Pull request by combspk**

### CODE REVIEW

File: app/agents/tools.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from langchain.base_language import BaseLanguageModel
 
 from .tp_tools import *
-from .s2 import SemanticSearch
```
Comment (boratyng) [inline app/agents/tools.py]: @combspk , you are removing changes from my last PR. Could you keep this  line in? Thanks!

File: app/agents/tools.py [hunk 2]
```diff
@@ -15,8 +15,7 @@ def make_tools(llm: BaseLanguageModel, verbose=True):
         SMILES2Weight(),
         FuncGroups(),
         ExplosiveCheck(),
-        ControlChemCheck(),
-        SemanticSearch()#,
```
Comment (boratyng) [inline app/agents/tools.py]: @combspk , you are removing changes from my last PR. Could you keep this  line in? Thanks!

**Review by boratyng — CHANGES_REQUESTED** [review]

File: app/agents/tools.py (hunk 1 above)
Comment (combspk) [reply app/agents/tools.py]: Sorry, should be resolved now, forgot to pull first

File: app/agents/tools.py (hunk 2 above)
Comment (combspk) [reply app/agents/tools.py]: Sorry, should be resolved now, forgot to pull first

**Review by boratyng — APPROVED** [review]

**Comment by combspk** [comment]
Closes #12

**Pull request merged**